vividus-analytics/src/main/java/org/vividus/analytics/GoogleAnalyticsFacade.java

```diff
@@ -83,13 +89,36 @@ private static String cid()
         return UUID.nameUUIDFromBytes(DigestUtils.sha512Hex(projectName).getBytes(UTF_8)).toString();
     }
 
-    public void postEvent(AnalyticsEvent analyticsEvent)
+    public void postEvent(AnalyticsEventBatch analyticsEvent)
+    {
+        List<String> encodedEvents = analyticsEvent.getEvents().stream()
+                .map(this::asParams)
+                .map(params -> URLEncodedUtils.format(params, UTF_8))
+                .collect(Collectors.toList());
+
+        if (encodedEvents.size() > MAX_EVENTS_PER_BATCH)
+        {
+            List<String> skippedEvents = encodedEvents.subList(MAX_EVENTS_PER_BATCH, encodedEvents.size());
```

Comment:
Looks strange to me, why do we need to limit statistics to 20? At the moment, when the integration tests already have 19 :) @valfirst

Reply:
20 is a limit provided by the GA measurement API, it was initially proposed to partition incoming events by 20 but we came to the conclusion that there is a very little chance that anyone have more than 20 plugins per project
cc @valfirst